dist/tools/zsh-completion/zsh-riot.sh

```diff
@@ -31,6 +31,43 @@ function _bools {
     _describe 'bool' _bool_vals
 }
 
+function _programmers {
+    local -a _programmer_vals=(
+        "openocd:use OpenOCD for programming via JTAG/SWD (default for most boards)"
+        "jlink:use Segger's JLinkExe for programming (requires original Segger programmers)"
+        "esptool:program ESP32 (S2,S3,C2,C3,C6,...) ESP8266 via UART"
+        "avrdude:default for AVR boards"
+        "lpc2k_pgm:default for lpc23xx boards"
+        "bossa:for SAM based Arduino boards"
+        "edbg:for SAM based evaluation board"
+        "nrfutil:for nRF5x boards with nRF bootloader"
+        "stm32flash:for STM32 boards via UART bootloader"
+        "uniflash:for CC13xx / CC26xx boards"
+        "cc2538-bsl:for CC2538 boards"
+        "mspdebug:for MSP430 boards"
+        "goodfet:for some MSP430 boards"
+        "uf2conf:program via UF2 bootloader (except RP2040)"
+        "elf2uf2:program via UF2 bootloader (RP2040)"
+        "cpy3remed:program via ST-Link using filesystem disk interface"
+        "adafruit-nrfutil:program via Adafruits nRF5x bootloader"
+    )
+
+    _describe 'programmer' _programmer_vals
+}
+
+function _hw_programmers {
+    local -a _hw_programmer_vals=(
+        "stlink:STMicroelectronics' ST-LINK/V2 or similar"
+        "jlink:Segger's J-Link programmer (e.g. J-Link Edu Mini)"
+        "dap:CMSIS DAP compatible programmer"
+        "ftdi:Bit-banging SWD/JTAG via FTDI chip"
+        "xds110:TI XDS110 programmer"
+        "sysfs_gpio:Bit-baning SWD/JTAG via GPIOs using the sysfs interface"
+    )
```

Comment:
There are some more in `makefiles/tools/openocd-adapters`:

```suggestion
        "buspirate:Bit-baning SWD/JTAG via Bus Pirate GPIOs"
        "iotlab:Debugging adapter used by the FIT IoT lab"
        "mulle:Mulle programmer board"
        "raspi:Bit-banging SWD/JTAG via GPIOs on the Raspberry Pi (direct register access)"
        "stlink-dap:STMicroelectronics' ST-LINK/V3 and ST-LINK/V2 (from firmware V2J24)"       
    )
```

Reply:
Oh, that one is also missing in the docs I think (or I overlooked it)